java/core/src/main/java/com/microsoft/bond/StructBondTypeResolver.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.bond;
+
+/**
+ * Responsible for resolving generic Bond struct types.
```

Comment:
Updating comments.

---
In reply to: [123373480](https://github.com/Microsoft/bond/pull/502#discussion_r123373480) [](ancestors = 123373480)